arch/risc-v/src/mpfs/mpfs_usb.c

```diff
@@ -228,6 +228,7 @@ static void   mpfs_epset_reset(struct mpfs_usbdev_s *priv, uint16_t epset);
  ****************************************************************************/
 
 static struct mpfs_usbdev_s g_usbd;
+static uint8_t g_clkrefs;
```

Comment:
Maybe this can be a part of `struct mpfs_usbdev_s` as an alternative to global variable?

Reply:
I thought of that but the same clock is used for all instances so the global counter is fine.

Reply:
Ok. Just wanted to confirm this decision.